FindMe/View Controllers/FindMeViewController.m

```diff
@@ -15,6 +15,7 @@ @interface FindMeViewController ()
 @property (strong, nonatomic) LocationManager *locationManager;
 @property (weak, nonatomic) IBOutlet UILabel *timeLabel;
 @property (strong, nonatomic) NSTimer *timer;
+@property (strong, nonatomic) NSDate *dateAndTimeNow;
```

Comment:
This doesn't make any sense to me. If it's a property then once you set it the name is immediately wrong. This has to be a function for it to be now. Also NSDate includes the time.... I am almost 100% sure you can delete this.